### PR TITLE
[Doc] add rising_wave.md to integrations

### PR DESCRIPTION
**Pull request by evelynzhaojie**
Signed-off-by: evelynzhaojie <everlyn.zhaojie@gmail.com

## What I'm doing:

1. add the Risingwave doc to Integrations
2. add a note for Hive Bitmap UDF in function list
3. fix chinese characters across English doc

## What type of PR is this:

- [ ] BugFix
- [ ] Feature
- [ ] Enhancement
- [ ] Refactor
- [ ] UT
- [x] Doc
- [ ] Tool

Does this PR entail a change in behavior?

- [ ] Yes, this PR will result in a change in behavior.
- [x] No, this PR will not result in a change in behavior.

If yes, please specify the type of change:

- [ ] Interface/UI changes: syntax, type conversion, expression evaluation, display information
- [ ] Parameter changes: default values, similar parameters but with different default values
- [ ] Policy changes: use new policy to replace old one, functionality automatically enabled
- [ ] Feature removed
- [ ] Miscellaneous: upgrade & downgrade compatibility, etc.

## Checklist:

- [ ] I have added test cases for my bug fix or my new feature
- [ ] This pr needs user documentation (for new or modified features or behaviors)
  - [ ] I have added documentation for my new feature or new function
- [ ] This is a backport pr

## Bugfix cherry-pick branch check:
- [x] I have checked the version labels which the pr will be auto-backported to the target branch
  - [x] 3.2
  - [x] 3.1
  - [ ] 3.0
  - [ ] 2.5

### CODE REVIEW

File: docs/zh/integrations/streaming/rising_wave.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
+---
+displayed_sidebar: "Chinese"
+---
+
+# **从 RisingWave 持续导入**
```
Comment (EsoragotoSpirit) [inline docs/zh/integrations/streaming/rising_wave.md:5]: Don't need bolding on title

File: docs/zh/integrations/streaming/rising_wave.md [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+| TIMESTAMP                                             | DATETIME       |
+| TIMESTAMP WITH TIME ZONE（建议预先转类型为TIMESTAMP）     | 不支持         |
+| INTERVAL（建议预先转类型为 VARCHAR）                      | 不支持         |
+| STRUCT                                                | JSON           |
+| ARRAY                                                 | ARRAY          |
+| BYTEA（建议预先转类型为 VARCHAR）                     | 不支持         |
+| JSONB                                                 | JSON           |
+| SERIAL                                                | BIGINT         |
+
+## **使用示例**
```
Comment (EsoragotoSpirit) [inline docs/zh/integrations/streaming/rising_wave.md:64]: Don't need bolding on title

File: docs/zh/integrations/streaming/rising_wave.md [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+| starrocks.user                                               | 用于访问 StarRocks 数据库的用户名。                          |
+| starrocks.password                                           | 用户名所对应的密码。                                         |
+| starrocks.database                                           | StarRocks 目标表所在的数据库名。                             |
+| starrocks.table                                              | StarRocks 目标表。                                           |
+| starrocks.partial_update                                     | （选填）是否启用部分列更新的优化。在更新列较少时，启用该选项可以提高导入性能。 |
+| type                                                         | 输出数据类型：<ul><li>`append-only`: 只会输出 Insert 操作。</li><li>`upsert`: 输出变更操作流。在此类型下，StarRocks 目标表必须为主键表。</li></ul>            |
+| force_append_only                                            | （选填）当输出类型为 append-only 而 Sink 实际有可能输出 upsert/delete 变更时，将 upsert 和 delete 数据丢弃，强行让 Sink 输出 append-only 流。 |
+| primary_key                                                  | （选填）StarRocks 表的主键。当 `type` 为 `upsert` 时，需要填入该项。 |
+
+## **数据类型映射**
```
Comment (EsoragotoSpirit) [inline docs/zh/integrations/streaming/rising_wave.md:41]: Don't need bolding on title

File: docs/zh/integrations/streaming/rising_wave.md [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+| INTEGER                                               | INT            |
+| BIGINT                                                | BIGINT         |
+| REAL                                                  | FLOAT          |
+| DOUBLE                                                | DOUBLE         |
+| DECIMAL                                               | DECIMAL        |
+| DATE                                                  | DATE           |
+| VARCHAR                                               | VARCHAR        |
+| TIME（建议预先转类型为 VARCHAR）                         | 不支持         |
+| TIMESTAMP                                             | DATETIME       |
+| TIMESTAMP WITH TIME ZONE（建议预先转类型为TIMESTAMP）     | 不支持         |
```
Comment (EsoragotoSpirit) [inline docs/zh/integrations/streaming/rising_wave.md:56]: 先转类型为TIMESTAMP
TIMESTAM 前空格

**Review by EsoragotoSpirit — APPROVED** [review]

**Review by amber-create — APPROVED** [review]

**Comment by evelynzhaojie** [comment]
backport to branch-3.1 #42226 42226

**Pull request merged**